pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -588,14 +590,27 @@ func (r *ReconcileSyncSetInstance) applySyncSetResources(ssi *hivev1.SyncSetInst
 
 	// determine if we can gather info for all resources
 	infos := []hiveresource.Info{}
+	unstructureds := []*unstructured.Unstructured{}
 	for i, resource := range resources {
+		unstructureds = append(unstructureds)
+		// Inject the hive managed annotation to help end-users see that a resource is managed by hive:
```

Comment:
This comment looks out of place here. Is it more appropriate to put it at line 685 in `applyResource` method?

Reply:
:+1: thanks